package/MDAnalysis/core/AtomGroup.py

```diff
@@ -685,6 +764,7 @@ def universe(self):
             return self._universe
 
     @universe.setter
+    @deprecate(message="{}; Atoms will not be able to leave their Universes.".format(_FIFTEEN_DEPRECATION))
```

Comment:
When a user tries to set the universe an ``Atom`` is a member of, this warning will appear. If you're asking where this is used in the codebase, no idea. In the new scheme though it doesn't make sense to do this.